mozilla_schema_generator/glean_ping.py

```diff
@@ -57,6 +57,22 @@ def __init__(self, repo, **kwargs):  # TODO: Make env-url optional
             **kwargs,
         )
 
+    def get_schema(self) -> Schema:
+        schema = super().get_schema()
+
+        # We need to inject placeholders for the url2, text2, etc. types as part
+        # of mitigation for https://bugzilla.mozilla.org/show_bug.cgi?id=1737656
+        for metric_name in ["labeled_rate", "jwe", "url", "text"]:
```

Comment:
I'm going to move this somewhere else. By putting it here, this change will cause the deployed JSON schemas for glean apps used to validation to contain these fields, which we don't want. See https://github.com/mozilla-services/mozilla-pipeline-schemas/commit/d07e70bfc445557c45f8016ddcbacb599557efda

Reply:
I've accomodated this by adding a `generic_schema` boolean parameter.